jha/src/views/enrolment/SpouseInfoPage.vue

```diff
@@ -1331,6 +1339,8 @@ export default {
         this.spousePreviousBCHealthNumber = null;
         this.spouseBeenReleasedFromInstitution = null;
         this.spouseDischargeDate = null;
+        this.spouseIncome = null;
```

Comment:
Can I ask for clarity about what this line does? It looks like it sets the field to null every time a user presses the continue button, unconditionally. Aren't there cases where we would want this data to remain and not be set to null?

Reply:
This is the `setEmptyFields` method. The values are getting set to `null` when the user chooses not to add a spouse.